compiler-core/src/format/tests.rs

```diff
@@ -1241,6 +1241,43 @@ fn expr_float() {
   -1_234_567.0
   -12_345_678.0
 }
+"#
+    );
+
+    assert_format_rewrite!(
+        r#"fn main() {
+  1.0e1
+  1.0e-1
+  -1.0e1
+  -1.0e-1
+  1.0e10
+  1.0e-10
+  -1.0e10
+  -11.0e-10
+  1.0e100
+  1.0e-100
+  -1.0e100
+  -11.0e-100
+  1.0000e100
+  1.0000e100_100
+}
+"#,
+        r#"fn main() {
+  1.0e1
+  1.0e-1
+  -1.0e1
+  -1.0e-1
+  1.0e10
+  1.0e-10
+  -1.0e10
+  -11.0e-10
+  1.0e100
+  1.0e-100
+  -1.0e100
+  -11.0e-100
+  1.0000e100
+  1.0000e100_100
+}
```

Comment:
What's the rewrite here? It looks the same to me?

Reply:
I think the idea is to show that there is no rewrite for the underscores and that it expected to stay the same?

Reply:
This macro is for rewrites, the `assert_format` macro is for testing formats otherwise. I wasn't sure which it was here

Reply:
My understanding is that it's to show that the formatter doesn't do anything here. If they're unneeded, I'm happy to remove!

Reply:
This macro is for rewrites, the `assert_format` macro is for testing formats otherwise. I wasn't sure which it was here